content/faq/36_where-to-host-projects/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: where can I host my projects?
+type: "faq"
+priority: "36"
+tags: ["projects", "5P"]
+---
+
+Host it on [netlify.com](https://www.netlify.com/).\
```

Comment:
list down few more places. This answer needs research. Take it out from the PR right now and put it as a blog.
10 places you can host your projects for free. AWS, Azure, Firebase, Netlfiy, Now.. Do a list with all links..

Reply:
Noted will do that.